### PR TITLE
Ember Times Issue 67

### PR DESCRIPTION
**Pull request by tylerturdenpants**
- [x] [Ember Support for Storybook](https://github.com/storybooks/storybook/pull/4237) (🔒 @chrisrng)
- [ ] http://embercamp.com/ talks, deep dive? https://github.com/ember-chicago/ember-camp-2018-notes (maybe wait for the talk videos to be published? Keep an eye on #ember-camp)
- [ ] This is just an idea, but I think GraphQL with Ember would be interesting https://emberfest.eu/schedule/#rocky-neurock
- [x] Hacktoberfest (:lock: @amyrlam)
  - [x] All of these addons at https://emberobserver.com/lists/invalid-repo-url need to be "found" on GitHub or the like and have the repo PR'd to the package.json, 977 issues, one for each of those addons aka the package.json for each of those addons (the repos need to be hunted down) need a PR to update the "repository" key (:lock: @amyrlam)
    - a good example of what needs to be done on the 977 repos https://github.com/ember-cli/ember-cli/blob/master/package.json#L25-L28
    - There's more info here: https://docs.npmjs.com/files/package.json#repository If a repo is on github, a shorthand can be used ala `"repository": "ember-cli/ember-cli"`
- [x] codesandbox.io ember support (:lock: @Alonski)
- [ ] ...

capture d ecran 2018-10-05 a 17 28 48

### CODE REVIEW

**Comment by mike-north** [comment]
we should be able to announce [codesandbox.io emberjs support](https://github.com/CompuIves/codesandbox-client/pull/1113) in this issue. I should have a blog article + some other information in the next ~2 days

**Comment by mike-north** [comment]
Here's the blog article for codesandbox support - https://medium.com/@mikenorth/ember-community-meet-codesandbox-10a43076b3fa

And here's the starter project for a "vanilla" ember app https://codesandbox.io/s/github/mike-north/ember-new-output/tree/vanilla

and for a typescript app
https://codesandbox.io/s/github/mike-north/ember-new-output/tree/ts

File: source/blog/2018-10-05-the-ember-times-issue-67.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+---
+
+## [Come Play in the Ember CodeSandbox 🏖️🏰](https://medium.com/@mikenorth/ember-community-meet-codesandbox-10a43076b3fa)
+
+[CodeSandbox](https://codesandbox.io) is an online code editor for the Web, and now for Ember as well! [@mike-north](https://github.com/mike-north) has done a lot of work to get Ember CLI working inside of CodeSandbox so that we can develop, showcase, and share live apps on the web! 
+
+What allowed this to happen is that CodeSandbox released server-side evaluated sandboxes, which now allows CLI Node apps to run and be served by the site. 
+
+Of course we want to give a shoutout to [@Guarav0](https://github.com/Gaurav0) and the rest of the [Ember Twiddle](https://ember-twiddle.com/) team for their **amazing** work. The way Ember Twiddle works is by [emulating](https://github.com/ember-cli/ember-twiddle/blob/ac116eb20d1e6c2152313f865159c443a9e2bd6f/app/services/ember-cli.js#L261-L295) a node environment. With CodeSandbox, we get **real Ember CLI** out of the box.
```
Comment (Gaurav0) [inline source/blog/2018-10-05-the-ember-times-issue-67.md]: My name still isn't fixed here.

Comment (amyrlam) [reply source/blog/2018-10-05-the-ember-times-issue-67.md]: sorry about that!

**Comment by tylerturdenpants** [comment]
Reviewed and LGTM

**Pull request merged**